cores/esp32/esp32-hal-spi.c

```diff
@@ -55,6 +55,10 @@
 #include "esp_intr.h"
 #endif
 
+#if ESP_IDF_VERSION_MAJOR >= 4 && ESP_IDF_VERSION_MINOR >= 3
```

Comment:
IDF version checks for v4.3 are not necessary on master, it will always be v4.4+.

Reply:
Will edit that thanks for the information didn't know that.